charlib/characterizer/cell.py

```
@@ -139,7 +139,34 @@ def match_direction(pin_name):
+            # Get variable names
+            if pin in [pair.inverting_port_name for pair in self.diff_pairs.values()]:
+                continue # skip inverting pins
+            storage_vars = [var,]
+            for pair in self.diff_pairs.values():
+                if pin == pair.noninverting_port_name:
+                    storage_vars.append(feedback_paths[pair.complement(pin)])
+            if len(storage_vars) < 2:
+                # No inverting output. Append inv to the end of the first var name
+                storage_vars.append(f'{var}inv')
+            var_string = ', '.join(storage_vars)
+            to_expr = lambda p: f'{p.name}{"\'" if p.is_inverted() else ""}'
```

Ruff (0.15.17)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `to_expr` as a `def`

(E731)